### PR TITLE
LPS-156394 change State to Task in My Workflow Tasks

### PR DESCRIPTION
**Pull request by M-IgorFranca**
Related: https://issues.liferay.com/browse/LPS-156394

### CODE REVIEW

**Comment by M-IgorFranca** [comment]
ci:test:sf

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path [hunk 1]
```diff
@@ -17,7 +17,7 @@
 </tr>
 <tr>
 	<td>STATE</td>
-	<td>//div/span[contains(.,'State')]/..//div[@class='panel-body']</td>
```
Comment (naraandrade) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path:20]: @M-IgorFranca I checked the macros that are using this path and both of them refer to the same screen (one of them in the view of a task via `Assigned to My Roles` and the other one in the view of a task via `Assigned to Me`). You can just replace 'State' with 'Task' :) Also, go ahead and change the name of the path to TASK as well.

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path (hunk 1 above)
Comment (M-IgorFranca) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/paths/MyWorkflowTasksTask.path:20]: Thanks for the review! I was unsure if I could make this change, so I upgraded to the safest option.

**Comment by M-IgorFranca** [comment]
ci:test:sf

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Review by naraandrade — CHANGES_REQUESTED** [review]
@M-IgorFranca You need to update all places that call `MyWorkflowTasksTask#STATE` to `MyWorkflowTasksTask#TASK`.

**Comment by M-IgorFranca** [comment]
ci:test:sf

**Comment by M-IgorFranca** [comment]
ci:test:relevant

**Comment by M-IgorFranca** [comment]
> @M-IgorFranca You need to update all places that call `MyWorkflowTasksTask#STATE` to `MyWorkflowTasksTask#TASK`.

Sorry @naraandrade, I forgot to update the macro. I already updated, I'm waiting for the relevant to request a new review, thanks.

**Comment by naraandrade** [comment]
> > @M-IgorFranca You need to update all places that call `MyWorkflowTasksTask#STATE` to `MyWorkflowTasksTask#TASK`.
> 
> Sorry @naraandrade, I forgot to update the macro. I already updated, I'm waiting for the relevant to request a new review, thanks.

No problem :) The last failures seem to be unrelated. I will just run the relevant again.

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**